generated/_task_modules/channels/ai_channel.py

```diff
@@ -20,7 +20,7 @@
     EddyCurrentProxProbeSensitivityUnits, ExcitationDCorAC,
     ExcitationIdleOutputBehavior, ExcitationSource,
     ExcitationVoltageOrCurrent, FilterResponse, FilterType,
-    ForceIEPESensorSensitivityUnits, ForceUnits, FrequencyUnits, Impedance1,
```

Comment:
`Impedance1` is no longer imported, which suggests that the properties that use it were deleted or changed.